### DISCUSSION TITLE
获取信用的会客室访问环节可能需要重新适配/做图像识别

### DISCUSSION THREAD

**Discussion by TimCrownie** [MaaAssistantArknights/MaaAssistantArknights#8531] [General]
因为不确定是否可以放在Issue里，所以先开个Discussion好了。
![Screenshot_20240309-031226](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/63519560/01ef8180-f9d0-4146-844b-076421affb44)
如图，好友页面的“好友列表”板块大改，给MAA带来的最大问题就是会客室访问环节已经无法正常进行，停留一会后会直接跳转去信用商店（未报错），但是由于我刚手滑删掉了log所以没得了（这也是我没发Issue而是发Discussion的另一个原因）
这个应该没太大工作量（相对于彩虹六号联动的主界面皮肤来说？），而且说实话已经习惯了让MAA代劳，所以一时间还是有点不太适应的。
另外有个不知道该不该发Issue的问题，即我这边MAA的某个配置会被疑似重开然后被排到窗口预览的最后一个（但是实际上仍然正常，除了看着膈应用着不顺手之外没啥问题）
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/63519560/188c7225-e863-4043-8193-5715cc124776)
PS. 图中的顺序为出了问题之后的，没出问题的时候是"α-EN α β γ Mi5-EN XZ1 Mi5"的排列

**Comment by ABA2396** [comment]
5.2.0-beta.2 fixed

**Reply by TimCrownie** [reply]
太好了，感谢我的神们让牛牛重获生机（？）
